traffic_portal/test/integration/README.md

```diff
@@ -1,30 +1,47 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
 # Traffic Portal Test Automation
+This directory contains integration tests for Traffic Portal.
 
-**Test Development Environment Setup**
+## Prerequisites
+* [Node](https://nodejs.org) v6 or later.
```

Comment:
We only support nodejs [v12+](https://traffic-control-cdn.readthedocs.io/en/latest/development/traffic_portal.html)